tests/test_plugins_os_unix_linux_debian_apt.py

```diff
@@ -23,3 +27,15 @@ def test_apt_logs(test_file, target_unix, fs_unix):
 
     for record in results:
         assert record.package_manager == "apt"
+
+    assert results[0].ts == dt(2022, 9, 2, 6, 36, 31, tzinfo=tz)
```

Comment:
I don't believe you need to specifically use the type from `flow.record` here, I think you can just use plain `datetime`.

Reply:
Fixed in [6d1aec6](https://github.com/fox-it/dissect.target/pull/128/commits/6d1aec642fa16c69950ff364c3c97f8ac820b371)